R/compare.R

```diff
@@ -142,11 +142,16 @@ order_proxy <- function(proxy, direction = "asc", na_value = "largest") {
   }
 
   if (is.data.frame(proxy)) {
-    args <- unname(proxy)
-    args$decreasing <- decreasing
-    args$na.last <- na.last
+    # Avoid empty data frames getting a `NULL` order
+    if (vec_size(proxy) == 0L) {
+      return(integer(0L))
+    }
 
-    do.call(base::order, args)
+    args <- unname(proxy)
+    order_expr <- expr(
```

Comment:
Wouldn't `exec()` be simpler here?

Reply:
Oh! Definitely. Great example of `exec()` being nicer than `do.call()`